src/main/java/gregtech/integration/jei/recipe/primitive/OreByProduct.java

```diff
@@ -110,6 +110,13 @@ public OreByProduct(Material material) {
         simpleWashers.add(new ItemStack(Items.CAULDRON));
         simpleWashers.add(MetaTileEntities.ORE_WASHER[GTValues.LV].getStackForm());
 
+        if (!material.hasProperty(PropertyKey.BLAST)) {
+            addToInputs(new ItemStack(Blocks.FURNACE));
```

Comment:
Should this be the vanilla furnace, or the electric furnace?

Reply:
I've had it on vanilla furnace since I wrote the page, I think it makes it more clear that direct smelting is a pre-LV way to process ores, similar to the simple washer cycling with the cauldron